test/hpe_docker_unit_test.py

```diff
@@ -94,6 +94,17 @@ def _mock_execute_api(self, mock_objects, plugin_api=''):
         req_body = self._get_request_body(self.get_request_params())
 
         _api = api.VolumePlugin(reactor, self._host_config, self._all_configs)
+        req_params = self.get_request_params()
+        backend = req_params.get('backend', 'DEFAULT')
+
+        import time
```

Comment:
Please move this import statement to the top section so that importing is avoid at every invocation of this funtion

Reply:
Thanks @imran-ansari , i will move the import statement